helm/tiled/templates/deployment.yaml

```diff
@@ -32,6 +32,28 @@ spec:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       containers:
         - name: {{ .Chart.Name }}
+          command:
+          - "python"
+          {{- if .Values.debug.enabled}}
+          - "-Xfrozen_modules=off"
+          - "-m"
+          - "debugpy"
+          {{- if .Values.debug.suspend }}
+          - "--wait-for-client"
+          {{- end }}
+          - "--listen"
+          - "0.0.0.0:{{ .Values.debug.port }}"
+          {{- end }}
+          - "-m"
+          args:
+          - "tiled"
+          - "serve"
+          - "config"
+          - "--host"
+          - "0.0.0.0"
+          - "--port"
+          - "8000"
+          - "--scalable"
```

Comment:
These duplicate the Dockerfile `CMD` arguments: Kubernetes uses `command` ~analogously to Docker `ENTRYPOINT` and `args` ~analogously to Docker `CMD`. Yes, it's frustrating. 

Unfortunately if you set `command`, `CMD` args are ignored, if they were picked up we wouldn't need to duplicate this here.